assets/js/admin/form-builder.js

```diff
@@ -1124,6 +1409,31 @@
 			field.closest( '.evf-admin-grid' ).find( '[data-field-id="' + old_key + '"]' ).after( newFieldCloned );
 			$(document).trigger('everest-form-cloned', [ new_key, field_type ]);
 			EVFPanelBuilder.switchToFieldOptionPanel(new_key);//switch to cloned field options
+
```

Comment:
We can just introduce a trigger here and execute code :)

Reply:
Resolved with bacf45179fd6783ab70ef3a3b3778e656b75203a .